### ISSUE TITLE
The use of the "war" plugin provokes "false positive" in the "deployWar" task.

### ISSUE THREAD

**Issue by oscarlv** [WASdev/ci.gradle#5]
I found that in the gradle project is being used the gradle plugin "war" to do the deploy and undeploy of the applications and the use of this plugin is provoking a "false positive" when you call the "deployWar" without specify the params for the deploy.

The params are:
*-destinationDir
-archiveName*

I mean, my build file looks like this:

*File: build.gradle*
```
war{
	destinationDir =file("resourcesToDeploy/")
	archiveName = 'sample.war'
}
```
If you set those params there is not any problem, the result would be successful and the deploy of the application will be done.

![correctsetup](https://cloud.githubusercontent.com/assets/6841943/6176769/09026476-b2c7-11e4-8fec-b65b7bca48ff.PNG)

But, if those params are not set in the build file the result of the "deployWar" task is also successful. Even if has not been specified any file to deploy,  so, I added the following line in the "deployWar" task
just to see which file is being deployed:

*File: Liberty.groovy*
````
project.task('deployWar') {
    doLast {
        println 'File to deploy:' + project.war.archivePath
    }
}
````
(archivePath is the result of concatenate "destinationDir " and "archiveName ") according to the "war" plugin documentation:
http://gradle.org/docs/current/dsl/org.gradle.api.tasks.bundling.War.html

If the parameters are not set:

*File: build.gradle*
```
war{
	//destinationDir =file("resourcesToDeploy/")
	//archiveName = 'sample.war'
}
```

The output (when the parameters weren't set) is this: 

![falsepositive](https://cloud.githubusercontent.com/assets/6841943/6176935/4134e4f8-b2c8-11e4-8a40-9ef21f0f18a8.PNG)

As you can see is looking for a war file with the same name of the project in a folder named "libs" which doesn't exist in my project. 

The structure of my proyect is:
```
-GradlePractices
    -resourcesToDeploy/
    wlp-anttasks-1.1-SNAPSHOT.jar
    liberty-gradle-plugin-1.0-SNAPSHOT.jar
    build.gradle
```

**Comment by oscarlv** [comment]
I've already did a pull request to solve this, please review #9.

**Comment by oscarlv** [comment]
This was solved with #9. Thanks.
